tutorials/pushing-metrics-logs-from-scw-instance/index.mdx

```diff
@@ -0,0 +1,134 @@
+---
+meta:
+  title: How to push metrics and logs from a Scaleway Instance
+  description: This page shows you how to push metrics and logs from a Scaleway Instance
+content:
+  h1: How to push metrics and logs from a Scaleway Instance
+  paragraph: This page shows you how to push metrics and logs from a Scaleway Instance
+tags: cockpit observability metrics logs instance
+categories:
+  - observability
+  - compute
+  - cockpit
+  - instances
+dates:
+  validation: 2023-06-20
+  posted: 2023-06-20
+---
+
+This page shows you how to push your metrics and logs from a [Scaleway Instance](/compute/instances/concepts/#instance).
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+ - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+ - You have activated your [Cockpit](/observability/cockpit/how-to/activate-cockpit)
+</Message>
+
+<Message type="important">
```

Comment:
You can also link to https://www.scaleway.com/en/docs/observability/cockpit/reference-content/understanding-cockpit-usage/ ?